src/openforms/submissions/models/submission_value_variable.py

```diff
@@ -157,14 +157,27 @@ def remove_variables(self, keys: list) -> None:
             if key in self._variables:
                 del self._variables[key]
 
-    def static_data(self) -> dict:
+    def _get_static_data(self, other_registry: VariablesRegistry | None = None):
+        return {
+            variable.key: variable.initial_value
+            for variable in get_static_variables(
+                submission=self.submission,
+                variables_registry=other_registry,
+            )
+        }
+
+    def get_static_data(
+        self, other_registry: VariablesRegistry | None = None
+    ) -> JSONObject:
+        # Enseure we do not accidentally cache the non-default registry
```

Comment:
If anything else needs update in the code let's fix the typo as well

Reply:
I'll aply the suggestions!